tdd/__init__.py

```diff
@@ -344,6 +340,9 @@ def generate():
                 f' etag="{get_collection_etag()}"'
             )
             return response
+        raise HTTPBadRequest(
+            "'format' parameter is not recognized. Must be in 'array' or 'collection' if defined."
```

Comment:
```suggestion
            "'format' parameter is not recognized. Must be 'array' or 'collection' if defined."
```

I am not a native speaker but the proposal above reads better to me